apps/backend/src/event/entity/event.entity.ts

```diff
@@ -1,7 +1,8 @@
-import { IsDate, IsNotEmpty, IsString, IsUUID } from 'class-validator';
+import { Color, Priority, Status } from '@prisma/client';
+import { IsDate, IsEnum, IsNotEmpty, IsString, IsUUID } from 'class-validator';
 
-export enum Priority {
-  NONE = 'NONE',
+/*export enum Relevance {
```

Comment:
Why is this commented out?

Reply:
We overlooked issue #17 , and it haven't been cleaned up since we found the solution in the last meeting. Those enums were unnecessary. I will clean it up.